vm/message/src/lib.rs

```diff
@@ -10,7 +10,13 @@ pub use signed_message::*;
 pub use unsigned_message::*;
 
 use address::Address;
+use cid::Cid;
+use encoding::{
+    de::{self, Deserializer},
+    ser::{self, Serializer},
+};
 use num_bigint::BigUint;
+use serde::Deserialize;
```

Comment:
nit: Use the one in `encoding::de::Deserialize` (Weird that the two are referenced in the same file, although they are basically reexports of the same thing)